pkg/k8s/watchers/cilium_flow_logging.go

```diff
@@ -0,0 +1,123 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
In general, we avoid creating new watchers in this package, and prefer using the Resource[T] framework.

I realize this small comment is actually a fair bit of work :-(

Reply:
ok, I will look into that